tools/find_device.h

```diff
@@ -0,0 +1,14 @@
+#ifndef BL_TOOLS_FIND_DEVICE_H
+#define Bl_TOOLS_FIND_DEVICE_H
+
+/* The path name of the matched /dev node */
+char dev_node[300];
+
+/**
+ * Scan the /dev nodes and match the right ACM devices
+ *
+ * Return: number of matched devices
+ */
+int scan(void);
```

Comment:
I'd let the caller pass in a buffer for writing to, and let the caller deal with ensuring it creates one.

```suggestion
int scan(char *buffer, size_t len);
```